File: matiasmasso/Mat.NET/DTO/DTOBalanceSaldo.cs
using System;

namespace DTO
{
    public class DTOBalanceSaldo : DTOPgcCta
    {
        public decimal CurrentDeb { get; set; }
        public decimal CurrentHab { get; set; }
        public decimal PreviousDeb { get; set; }
        public decimal PreviousHab { get; set; }
        public DTOContact Contact { get; set; }

        public DTOBalanceSaldo() : base()
        {
        }

        public DTOBalanceSaldo(Guid oGuid) : base(oGuid)
        {
        }

        public bool IsDeutor()
        {
            return CurrentDeb > CurrentHab;
        }

        public bool IsCreditor()
        {
            return CurrentHab > CurrentDeb;
        }
    }
}

File: matiasmasso/Mat.NET/DT2/DTOBalanceSaldo.vb
Public Class DTOBalanceSaldo
    Inherits DTOPgcCta
    Property CurrentDeb As Decimal
    Property CurrentHab As Decimal
    Property PreviousDeb As Decimal
    Property PreviousHab As Decimal
    Property Contact As DTOContact

    Public Sub New()
        MyBase.New()
    End Sub

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
    End Sub

    Public Function IsDeutor() As Boolean
        Return CurrentDeb > CurrentHab
    End Function

    Public Function IsCreditor() As Boolean
        Return CurrentHab > CurrentDeb
    End Function
End Class
